src/components/TableRowComponent.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { Table } from 'semantic-ui-react';
 import { withRouter } from 'react-router-dom';
 
-export class TableRowComponent extends React.Component {
+class TableRowComponent extends React.Component {
```

Comment:
Change back to `export` as the normal export is there for tests. We don't want to test the default export which is wrapped in `withRouter`